Geofencing/app/src/main/AndroidManifest.xml

```diff
@@ -15,29 +14,37 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.google.android.gms.location.sample.geofencing" >
+    package="com.google.android.gms.location.sample.geofencing">
 
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
 
```

Comment:
When running on Android O, the JobScheduler will take care of wake locks but on previous versions of the platform you must request the WAKE_LOCK permission.